RoaringBitmap/src/main/java/org/roaringbitmap/FastAggregation.java

```diff
@@ -445,10 +445,7 @@ private static int workShyAndCardinality(RoaringBitmap... bitmaps) {
           continue;
         }
         Container container = bitmap.highLowContainer.getContainerAtIndex(index);
-        Container and = tmp.iand(container);
```

Comment:
This is an optimisation

Reply:
this increases throughput when G1 is the active GC by avoiding its write barrier, which can be surprisingly expensive in tight loops

Reply:
Ok. Please see https://github.com/RoaringBitmap/RoaringBitmap/pull/672

Reply:
Sorry, I did not realize that. I rolled back all related changes.